src/muler/igrins.py

```diff
@@ -42,15 +42,19 @@ class IGRINSSpectrum(EchelleSpectrum):
     A container for IGRINS spectra
 
     Args:
-        file (str): A path to a reduced IGRINS spectrum from plp
+        file (str): A path to a reduced IGRINS spectrum from plp of file type .spec.fits
+            or .spec_a0v.fits.
+        wavefile (str):  A path to a reduced IGRINS spectrum storing the wavelength solution
+            of file type .wave.fits.
         order (int): which spectral order to read
         cached_hdus (list) :
-            List of two fits HDUs, one for the spec_a0v.fits, and one for the
-            sn.fits file, to reduce file I/O for multiorder access.
-            If provided, must give both HDUs.  Optional, default is None.
+            List of two or three fits HDUs, one for the spec.fits/spec_a0v.fits, one for the
+            sn.fits file, and one optional one for the .wave.fits file
+            to reduce file I/O for multiorder access.
+            If provided, must give both (or three) HDUs.  Optional, default is None.
     """
 
-    def __init__(self, *args, file=None, order=10, cached_hdus=None, **kwargs):
+    def __init__(self, *args, file=None, wavefile=None, order=10, cached_hdus=None, **kwargs):
```

Comment:
I'm slightly inclined to move `wavefile=None` later in the kwargs list.  Most stellar spectroscopists will never need it, and so pushing it later in the list makes it less like to be confused an expected input rather than an optional one.

Reply:
Doesn't matter to me really.  Although my understanding is that the .spec.fits files and .wave.fits files are supposed to be the main outputs of the IGRINS pipeline and that the .spec_a0v.fits files were an attempt to try to incorporate a crude telluric correction and relative flux calibration into the pipeline itself.  Are people really grabbing the .spec_a0v.fits for their science work nowadays?

Reply:
Moved wavefile to the end.